locale/es.json

```diff
@@ -5,11 +5,11 @@
     },
     "restarter": {
         "start_timeout": "Se ha agotado el tiempo de espera para el arranque.",
-        "crash_detected": "Caída del servidor detectada.",
```

Comment:
Why did you remove the accent from caida and the dot at the end?

Reply:
Maybe. typo and bad review. thx for fixing